src/module/actor/SR5Actor.ts

```diff
@@ -125,69 +66,147 @@ export class SR5Actor extends Actor {
         matrix.name = '';
         matrix.device = '';
 
-        // PARSE WEAPONS AND SET VALUES AS NEEDED
-        for (let item of Object.values(items)) {
-            const itemData: ItemData = (item.data as unknown) as ItemData;
-
-            const equipped = itemData.technology?.equipped;
-            if (equipped) {
-                if (itemData.armor && itemData.armor.value) {
-                    // if it's a mod, add to the mod field
-                    if (itemData.armor.mod) {
-                        armor.mod[item.name] = itemData.armor.value;
-                    } // if not a mod, set armor.value to the items value
-                    else {
-                        armor.base = itemData.armor.value;
-                        armor.label = item.name;
-                        for (const element of Object.keys(CONFIG.SR5.elementTypes)) {
-                            armor[element] = itemData.armor[element];
-                        }
+        // get the first equipped device, we don't care if they have more equipped -- it shouldn't happen
+        const device = items.filter((item) => item.isEquipped() && item.isDevice())[0];
```

Comment:
Use .find to get the first
```suggestion
        const device = items.find((item) => item.isEquipped() && item.isDevice());
```